lib/chef/resource/osx_profile.rb

```diff
@@ -24,13 +24,11 @@ class Resource
     # on the macOS platform. The osx_profile resource installs profiles by using
     # the uuidgen library to generate a unique ProfileUUID, and then using the
     # profiles command to install the profile on the system.
-    #
-    # @since 12.7
     class OsxProfile < Chef::Resource
       provides :osx_profile, os: "darwin"
       provides :osx_config_profile, os: "darwin"
 
-      identity_attr :profile_name
```

Comment:
This came along for the ride, but it doesn't need to be here anymore since we set name_property on this attribute anyways